game.py

```diff
@@ -1,12 +1,12 @@
-import datetime
 import random
-import sys
 from time import sleep
 
 import pygame
 
 from astar import astar
+from buttons_class import Buttons
 from coloredRect import ColoredRect
+from stats import STATS
```

Comment:
Class name should not be with capitals that is the convention for GLOBAL variables